### PR TITLE
Dotnet code coverage

### PR DESCRIPTION
**Pull request by fundthmcalculus**

### CODE REVIEW

**Review by tmarkovski — APPROVED** [review]

**Comment by tmarkovski** [comment]
Why does the test workflow exit with code 1 when all tests have passed?

**Comment by fundthmcalculus** [comment]
Because of the xamarinios10 build target that `dotnet test` does not support.

**Comment by tmarkovski** [comment]
> Because of the xamarinios10 build target that `dotnet test` does not support.

Could we switch the agent to `macos-latest`, run `msbuild` before the test step, and add `--no-build` to the test step? It seems to work fine locally for me.

**Comment by tmarkovski** [comment]
https://github.com/microsoft/setup-msbuild

File: .github/workflows/build-libs.yml [hunk 1]
```diff
@@ -115,6 +115,31 @@ jobs:
...
+    needs: build_macos
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - uses: actions/download-artifact@v2
+        with:
+          path: ./libs
+      - name: Build .net
+        run: msbuild /p:Configuration=Debug
+        working-directory: ./dotnet/Library
```
Comment (tmarkovski) [inline .github/workflows/build-libs.yml:131]: Just `working-directory: ./dotnet`

**Pull request merged**